docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
+
+The Faker connector generates realistically looking random data matching a
+specific schema.
+
+## Configuration
+
+To configure the Faker connector, create a catalog properties file
+`etc/catalog/example.properties` with the following contents:
+
+```text
+connector.name=faker
+```
+
+## Examples
+
+Faker generates data when reading from a table created in a catalog using this
+connector. This makes it easy to fill an existing schema with random data, by
+copying only the schema into a Faker catalog, and inserting the data back into
+the original tables.
+
+```sql
+CREATE TABLE faker.default.customer (LIKE production.public.customer EXCLUDING PROPERTIES);
+
+INSERT INTO production.public.customers
+SELECT *
+FROM faker.default.customers
+WHERE
+  born_at BETWEEN CURRENT_DATE - INTERVAL '150' YEAR AND CURRENT_DATE
+  AND age_years BETWEEN 0 AND 150
+LIMIT 100;
+```
+
+To generate more realistic data, choose specific generators by setting the
+`generator` property on columns:
+
+```sql
+SHOW CREATE TABLE production.public.customers;
+
+-- copy the output of the above query and add some properties:
+CREATE TABLE faker.default.customer (
+  id UUID NOT NULL,
+  name VARCHAR NOT NULL WITH (generator = '#{Name.first_name} #{Name.last_name}'),
+  address VARCHAR NOT NULL WITH (generator = '#{Address.fullAddress}'),
+  born_at DATE,
+  age_years INTEGER
+);
+```
+
+See the Datafaker's documentation for more information about
```

Comment:
We should probaly link to the datafaker.net site in the intro on the top of the docs

Reply:
also .. we should add data faker to the trino.io website and trino ot the datafaker website .. do you want to work on that @nineinchnick ?

Reply:
The Datafaker usage is optional in this connector - it's only used for character based data types, and only when a generator is chosen explicitly.

I'll definitely reach out to the author of Datafaker after 464 is out.

Reply:
> The Datafaker usage is optional in this connector - it's only used for character based data types, and only when a generator is chosen explicitly.

Lets explain that in the intro. 

> I'll definitely reach out to the author of Datafaker after 464 is out, and these docs are published, since they explain the usage well.

Great.. I can help on getting it onto the trino website